streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/bean/SenderEmailTest.java

```diff
@@ -77,7 +77,7 @@ void alert() {
       mail.setStatus(appState.name());
 
       StringWriter writer = new StringWriter();
-      Map<String, AlertTemplate> out = new HashMap<String, AlertTemplate>();
+      Map<String, AlertTemplate> out = new HashMap<>();
```

Comment:
ditto this test fails locally

... we just swallow the exception below.

Reply:
Because it is necessary to test the send email. It is necessary to manually fill in info such as sending emails, password ... These are private info and cannot be provided

Reply:
I suggest remove these testcase classes